doc/mgr/dashboard.rst

```diff
@@ -1070,3 +1070,178 @@ and loosely coupled fashion.
 
 .. include:: dashboard_plugins/feature_toggles.inc.rst
 .. include:: dashboard_plugins/debug.inc.rst
+
+
+Troubleshooting the Ceph Dashboard
```

Comment:
Since this section is now part of the Ceph Dashboard chapter, we can probably trim this to:
```suggestion
Troubleshooting
```